projects/tuist/fixtures/ios_app_with_intents/Sources/AppDelegate.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+    func application(
+        _ application: UIApplication,
+        didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil
+    ) -> Bool {        
+        window = UIWindow(frame: UIScreen.main.bounds)
+        
+        let viewController = UIViewController()
+        viewController.view.backgroundColor = .white
+        window?.rootViewController = viewController
+        window?.makeKeyAndVisible()
+
+        let model = MyModel()
+        model.save()
+        model.load()
```

Comment:
Does this compile?

For this fixture, it would be handy to import Intents and used one of the codegened intent types.

Reply:
Yes it compiles actually i copied this folder, but let me update this code to use intent